### ISSUE TITLE
Linters for Rust?

### ISSUE THREAD

**Issue by aergonaut** [houndci/hound#1344]
Is there interest in adding support for Rust linters to Hound? The obvious ones would be [Clippy](https://github.com/Manishearth/rust-clippy) and maybe [cargo-check](https://github.com/rsolomo/cargo-check).

I'd be interested in taking this on if it sounds like something that would be a good idea.

**Comment by gylaz** [comment]
Sounds like it could be useful. Is there a Heroku buildpack for Rust? Would we need to compile the linters first?

**Comment by aergonaut** [comment]
Yes, there is a Heroku buildpack: https://github.com/emk/heroku-buildpack-rust

And yeah, we would have to compile the linters

**Comment by gylaz** [comment]
Cool. If you'd like to lay down some work on this, we'd greatly appreciate it!
